crates/re_arrow_store/benches/arrow2_convert.rs

```diff
@@ -0,0 +1,130 @@
+//! Keeping track of performance issues/regressions in `arrow2_convert` that directly affect us.
+
+#[global_allocator]
+static GLOBAL: mimalloc::MiMalloc = mimalloc::MiMalloc;
+
+use arrow2::{array::PrimitiveArray, datatypes::PhysicalType, types::PrimitiveType};
+use criterion::{criterion_group, criterion_main, Criterion};
+use re_log_types::{
+    component_types::InstanceKey, external::arrow2_convert::deserialize::TryIntoCollection,
+    Component as _, DataCell,
+};
+
+// ---
+
+criterion_group!(benches, serialize, deserialize);
+criterion_main!(benches);
+
+// ---
+
+#[cfg(not(debug_assertions))]
+const NUM_INSTANCES: usize = 100_000;
+
+// `cargo test` also runs the benchmark setup code, so make sure they run quickly:
+#[cfg(debug_assertions)]
+const NUM_INSTANCES: usize = 1;
```

Comment:
that makes it a bit annoying though when one comes in with the intention of comparing debug & release builds :/

Reply:
Further investigations:
- `cargo test --bench` runs one iteration of each benchmark, not just the setup code (see https://github.com/rust-lang/cargo/issues/6445 and try it out with `cargo test -p re_arrow_store --bench '*'`), so we still need that thing.
- There's no way to pass `--debug` to `cargo bench`, you have to modify `RUSTFLAGS` which is an awful experience anyway, so...